src/shared/components/CategoryPicker/CategoryPicker.style.ts

```diff
@@ -4,8 +4,20 @@ import ToggleButton from '../ToggleButton'
 import { sizes, colors } from '@/shared/theme'
 
 export const Container = styled.div`
-  display: flex;
-  flex-wrap: wrap;
+  display: -webkit-box;
+  white-space: nowrap;
+  overflow: scroll;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+  > * {
+    &:first-child {
+      margin-left: var(--global-horizontal-padding);
+    }
+    &:last-child {
+      margin-right: var(--global-horizontal-padding);
+    }
+  }
 `
 
 export const StyledPlaceholder = styled(Placeholder)`
```

Comment:
We can adjust those styles now. All the items used to have a bottom margin to handle row wrapping. Now that all the items will reside in one row, we can get rid of that margin for items. At the same time we will need to update the padding for `StyledCategoryPicker` which purposefully sets a lower bottom padding because it takes this margin into account.